PyInstaller/building/build_main.py

```diff
@@ -160,6 +160,9 @@ def __init__(self, scripts, pathex=None, binaries=None, datas=None,
         runtime_hooks
                 An optional list of scripts to use as users' runtime hooks. Specified
                 as file names.
+        cipher
+                Ad optional instance of the pyz_crypto.PyiBlockCipher class
```

Comment:
Typo (will wait for the CI to complete at least once before committing again)
```suggestion
                Add optional instance of the pyz_crypto.PyiBlockCipher class
```

Reply:
CI build has finished. You can commit this now. Looks good btw.

Reply:
I'll rebase and push as soon as possibile!